src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx

```diff
@@ -26,6 +26,7 @@ interface Props {
 export default function QuestionsScreen(props: Props) {
   const [selected, setSelected] = useState<number | null>(null)
   const [questions, setQuestions] = useState<Array<Question>>([])
+  let questionToRemove = 0
```

Comment:
make this a state variable

Reply:
when I try to use setState for it, its read as undefined when passed in the body of the delete route, I don't know how to make it work

Reply:
just get rid of this altogether and refer to below suggestion